test/common/common/utility_test.cc

```diff
@@ -802,34 +802,42 @@ TEST(DateFormatter, FromTimeSameWildcard) {
 
 TEST(TrieLookupTable, AddItems) {
   TrieLookupTable<const char*> trie;
-  EXPECT_TRUE(trie.add("foo", "a"));
-  EXPECT_TRUE(trie.add("bar", "b"));
-  EXPECT_EQ("a", trie.find("foo"));
-  EXPECT_EQ("b", trie.find("bar"));
+  const char* cstr_a = "a";
+  const char* cstr_b = "b";
+  const char* cstr_c = "c";
+
+  EXPECT_TRUE(trie.add("foo", cstr_a));
+  EXPECT_TRUE(trie.add("bar", cstr_b));
+  EXPECT_EQ(cstr_a, trie.find("foo"));
```

Comment:
Just use EXPECT_STREQ here and below.

Reply:
We can do that, @mattklein123 . But if what you are testing in trie is that the strings passed to trie are stored as referenced (and not copied), then the test above is actually more correct than testing for string value equality. Your call.

Reply:
Ah OK makes sense, thanks.